docs/reference/query-dsl/rule-query.asciidoc

```diff
@@ -0,0 +1,77 @@
+[role="xpack"]
+[[query-dsl-rule-query]]
+=== Rule Query
```

Comment:
```suggestion
=== Rule query
++++
<titleabbrev>Rule</titleabbrev>
++++
```

Reply:
In the navigation, we don't show the word "query" for the other queries. The `titleabbrev` enables us to do that.